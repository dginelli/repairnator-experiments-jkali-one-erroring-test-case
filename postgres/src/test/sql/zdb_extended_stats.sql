select * from zdb_extended_stats('so_posts', 'answer_count', 'beer');