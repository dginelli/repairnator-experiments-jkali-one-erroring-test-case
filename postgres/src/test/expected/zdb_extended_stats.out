select * from zdb_extended_stats('so_posts', 'answer_count', 'beer');
 count | total | min | max | mean | sum_of_squares |     variance     |  std_deviation  
-------+-------+-----+-----+------+----------------+------------------+-----------------
    21 |    84 |   1 |  14 |    4 |            590 | 12.0952380952381 | 3.4778208831448
(1 row)

